DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -30,10 +30,10 @@ public void Initial_menu_should_have_name()
             var initialMenuViewModel = new InitialMenuViewModel(expectedCourseContent);
 
             // Then
-            initialMenuViewModel.Title.Should().Be($"{applicationName} - {customisationName}");
-            initialMenuViewModel.AverageDuration.Should().Be(averageDuration);
-            initialMenuViewModel.CentreName.Should().Be(centreName);
-            initialMenuViewModel.BannerText.Should().Be(bannerText);
+            initialMenuViewModel.CourseContent.Title.Should().Be($"{applicationName} - {customisationName}");
+            initialMenuViewModel.CourseContent.AverageDuration.Should().Be(averageDuration);
+            initialMenuViewModel.CourseContent.CentreName.Should().Be(centreName);
+            initialMenuViewModel.CourseContent.BannerText.Should().Be(bannerText);

```

Comment:
You can just check that the course content is equal to the expected course content 🙂

Reply:
Ah yes! Done in 8fbaf7c 🙂